### PR TITLE
Render documentation

### LINKED ISSUE #67: Criar estrutura que permite renderizar a documentação
### WHY

Precisamos garantir que seja possivel ver a documentação na rota `docs`

### HOW

Crie uma estrutura de "event listener" no arquivo do index.php, estabelecendo uma estrutura alternativa a aplicação padrão, ou seja, redirecionando o fluxo da aplicação atual

### STORY USER

Ao acessar o endereço `/docs` o usuário deverá ver a documentação da API do Mapas e tbm os endpoints que envolvem eventos

### CHECKLIST
- [x] Criar a rota e retornar a documentação (`/docs`)

### PR DESCRIPTION
**Pull request by henriquelopeslima**
Resolve #67

### CODE REVIEW

File: src/app/routes/api.php [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?php
+
+declare(strict_types=1);
+
+$api = [
```
Comment (alessandrofeitoza) [inline src/app/routes/api.php]: `return [`

File: src/app/routes/routes.php [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<?php
+
+declare(strict_types=1);
+
+require_once 'api.php';
```
Comment (alessandrofeitoza) [inline src/app/routes/routes.php]: Think about this:

```php
$web = require_once 'web.php';
$api = require_once 'api.php';

return [
   ...$api,
   ...$web,
];
```

File: public/index.php [hunk 3]
```diff
@@ -1,4 +1,20 @@
 <?php
+
+declare(strict_types=1);
+
 require_once 'bootstrap.php';
+require_once __DIR__.'/../src/app/routes/routes.php';
+
+$routes = $api ?? [];
```
Comment (alessandrofeitoza) [inline public/index.php]:
```php

$routes = require_once __DIR__.'/../src/app/routes/routes.php';
```

File: composer.json [hunk 4]
```diff
@@ -32,7 +32,8 @@
         "psr-4": {
             "MapasCulturais\\": "src/core",
             "MapasCulturais\\Modules\\": "src/modules",
-            "MapasCulturais\\Themes\\": "src/themes"
+            "MapasCulturais\\Themes\\": "src/themes",
+            "App\\": "src/app"
```
Comment (alessandrofeitoza) [inline composer.json]: Modify the order, to respect ASC sequence

**Review by alessandrofeitoza — APPROVED** [review]

**Review by TalysonSoares — APPROVED** [review]

**Pull request merged**